### ISSUE TITLE
Bug: Inserting and Removing mixtures can bug a vat

### ISSUE THREAD

**Issue by TarnishedArgent** [petrolpark/Destroy#146]
When adding a mixture and then removing it will sometimes bug the vat and delete all mixtures and gasses from the vat showing it as empty. Have to remake vat to fix it, Hard to reproduce.

**Comment by petrolpark** [comment]
I'm not sure what the issue is. If you remove the Mixture in a Vat it will become empty. Is the issue that it instantly voids the entire Vat?

**Comment by TarnishedArgent** [comment]
> I'm not sure what the issue is. If you remove the Mixture in a Vat it will become empty. Is the issue that it instantly voids the entire Vat?

Yes, it voids the whole vat and bugs it so you can't put anything in the vat at all.

**Comment by TarnishedArgent** [comment]
> > I'm not sure what the issue is. If you remove the Mixture in a Vat it will become empty. Is the issue that it instantly voids the entire Vat?
> 
> Yes, it voids the whole vat and bugs it so you can't put anything in the vat at all.

Haven't been able to reproduce but it's happened 3 times at this point

**Comment by petrolpark** [comment]
I think this got fixed.
